src/datomic_rest_api/get_handler.clj

```diff
@@ -37,40 +39,10 @@
-;;     (GET "/rest/widget/gene/:id/interactions" {params :params}
-;;         (get-interactions "gene" db (:id params) (str "rest/widget/gene/" (:id params) "/interactions"))) ;; needed work on nodes all - not quite lining up
-;;     (GET "/rest/field/gene/:id/interaction_details" {params :params}
-;;         (get-interaction-details "gene" db (:id params) (str "rest/field/gene/" (:id params) "/interaction_details"))) ;; wormbase is missing data section: why?
-     (GET "/rest/widget/gene/:id/mapping_data" {params :params}
-         (gene/mapping-data db (:id params) (str "rest/widget/gene/" (:id params) "/mapping_data")))
-;;     (GET "/rest/widget/gene/:id/human_diseases" {params :params}
-;;         (gene/human-diseases db (:id params) (str "rest/widget/gene/" (:id params) "/human_disease")))
-;;     (GET "/rest/widget/gene/:id/references" {params :params}
-;;         (get-references "gene" db (:id params) (str "rest/widget/gene/" (:id params) "/references")))
-;;     (GET "/rest/widget/gene/:id/reagents" {params :params}
-;;         (gene/reagents db (:id params) (str "rest/widget/gene/" (:id params) "/reagents"))) ;; looks correct; needs sort to confirm
-;;     (GET "/rest/widget/gene/:id/gene_ontology" {params :params}
-;;         (gene/gene-ontology db (:id params) (str "rest/widget/gene/" (:id params) "/gene_ontology"))) ;; substancially same structure. not producing the same results
-;;     (GET "/rest/widget/gene/:id/expression" {params :params}
-;;         (gene/expression db (:id params) (str "rest/widget/gene/" (:id params) "/expression"))) ;; This one is predominantly done but needs a little checking and sequence data
-;;     (GET "/rest/widget/gene/:id/homology" {params :params}
-;;         (gene/homology db (:id params) (str "rest/widget/gene/" (:id params) "/homology"))) ;; need to wait for homology data to be added to datomic database
-;;     (GET "/rest/widget/gene/:id/sequences" {params :params}
-;;         (gene/sequences db (:id params) (str "rest/widget/gene/" (:id params) "/sequences")))
-;;     (GET "/rest/widget/gene/:id/feature" {params :params}
-;;         (gene/features db (:id params) (str "rest/widget/gene/" (:id params) "/feature"))) ;; has a few values missing for gbrowse - not sure if needed or possible to get out of datomic
-     (GET "/rest/widget/gene/:id/genetics" {params :params}
-         (gene/genetics db (:id params) (str "rest/widget/gene/" (:id params) "/genetics")))
-     (GET "/rest/widget/gene/:id/external_links" {params :params}
-          (gene/external-links db (:id params) (str "rest/widget/gene/" (:id params) "/external_links")))
-     (GET "/rest/field/:class/:id/:field" [class id field]
-          (handle-field-get db class id field))))
+     (GET "/rest/widget/:class/:id/:widget" [class id widget :as request]
+          (handle-widget-get db class id widget request))
+     (GET "/rest/field/:class/:id/:field" [class id field :as request]
```

Comment:
Either `schema-name` or `ns-name` works great for me, and in fact is more technically correct.
fwiw, `ident` is the datomic concept of "identity" of an entity. 

e.g: if you have a gene entity/entity map, then:
 ```clojure
`(= (:db/ident gene) :gene/id)
```
holds true.
<<ns-name/schema-name>>/id is equvilant to the "name"/"id" of an ACeDB object.

[1]: http://docs.datomic.com/identity.html